packages/grafana-ui/src/utils/displayValue.ts

```diff
@@ -3,37 +3,12 @@ import _ from 'lodash';
 import moment from 'moment';
 
 // Utils
-import { getValueFormat, DecimalCount } from './valueFormats/valueFormats';
+import { getValueFormat } from './valueFormats/valueFormats';
 import { getMappedValue } from './valueMappings';
 import { getColorFromHexRgbOrName } from './namedColorsPalette';
 
 // Types
-import { GrafanaTheme, GrafanaThemeType, ValueMapping, Threshold } from '../types';
-
-export interface DisplayValue {
-  text: string; // Show in the UI
-  numeric: number; // Use isNaN to check if it is a real number
-  color?: string; // color based on configs or Threshold
-}
-
-export interface DisplayValueOptions {
```

Comment:
not sure this type should be moved, it's the arguments type to this display processor function, feels strange to move it.